modules/xfem/test/tests/single_var_constraint_2d/tests

```diff
@@ -55,5 +73,19 @@
     # XFEM requires --enable-unique-ids in libmesh
     unique_id = true
     abs_zero = 1e-8
+    requirement = 'MOOSE shall provide an ability to apply nonzero-jump nonzero-flux-jump constraint on a stationary interface for one variable.'
+    design = 'XFEMSingleVariableConstraint.md'
+    issues = '#6572'
+  [../]
+  [./equal_value]
+    type = Exodiff
+    input = equal_value.i
+    exodiff = 'equal_value_out.e'
+    map = false
+    # XFEM requires --enable-unique-ids in libmesh
+    unique_id = true
+    requirement = 'MOOSE shall provide an ability to two sided equal value constraint on a stationary interface for one variable.'
```

Comment:
The XFEM module shall provide an ability to apply a two-sided equal value constraint on a stationary interface for one variable.